metron-platform/metron-pcap/src/main/java/org/apache/metron/pcap/filter/PcapFilters.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.metron.pcap.filter;
+
+import org.apache.metron.pcap.filter.fixed.FixedPcapFilter;
+import org.apache.metron.pcap.filter.query.QueryPcapFilter;
+
+import java.util.function.Function;
+
+public enum PcapFilters {
+  FIXED(x -> new FixedPcapFilter()),
+  QUERY(x -> new QueryPcapFilter());
+
+  Function<Void, PcapFilter> filter;
+
+  PcapFilters(Function<Void, PcapFilter> filter) {
```

Comment:
It'd be cool if this was an actual interface like, Creator<T> { T create(); } and it lived in metron-common, as opposed to Function<Void, T>.  I bet we have use for this pattern elsewhere.